snapcraft/internal/elf.py

```diff
@@ -74,7 +71,6 @@ def ldd(path: str, ld_library_paths: List[str]) -> Dict[str, str]:
         ldd_lines = (
             subprocess.check_output(["ldd", path], env=env).decode().splitlines()
         )
-        logger.debug(f"ldd output:\n{ldd_lines}")
```

Comment:
If removing this, at least log things that don't match on https://github.com/snapcore/snapcraft/pull/3287/files#diff-f7cf307285f042f35434bbe680606c77R91 but skip logging for the usual suspects, we might fall into a parsing logic error and this really helped in the past.

Reply:
Warn about failed matches if unexpected.